index.html

```diff
@@ -244,36 +258,157 @@ <h1 class="title">About Myself</h1>
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-width="2"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    class="feather feather-chevron-down"
+                  >
+                    <polyline points="6 9 12 15 18 9"></polyline>
+                  </svg>
                 </label>
               </summary>
               <div class="summary-chevron-down">
-                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-up"><polyline points="18 15 12 9 6 15"></polyline></svg>
-                </div>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  stroke="currentColor"
+                  stroke-width="2"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  class="feather feather-chevron-up"
+                >
+                  <polyline points="18 15 12 9 6 15"></polyline>
+                </svg>
+              </div>
             </details>
           </div>
         </div>
         <!--End of Languages Article-->
       </div>
     </section>
     <!--End of About myself section-->
+
+    <!--The begin Contact form section-->
+    <footer id="Footer">
+      <div class="back-contact"></div>
+      <div class="contact-back">
+        <div class="background-contact">
+          <div class="container">
+            <div class="back-c"></div>
+            <div class="content-contact">
+              <div class="Contact-head">
+                <h2 class="title-contact">Contact me</h2>
+                <p class="text-contact">
+                  If you have an application you are interested in developing, a
+                  feature that you need built or a project that needs coding.
+                  I'd love to help with it
+                </p>
+              </div>
+              <form class="form-contact" action="https://formspree.io/f/mgedpajj" method="post" autocomplete="off">
+                <div class="input-warpper">
+                  <input
+                    class="input"
+                    type="text"
+                    placeholder="Full name"
+                    required
```

Comment:
- [ ] Kindly add `maxlength="30"` attribute to `name ` input as it is in the project requirements.